### PR TITLE
Defined get_mac_address_table() for JunOS, IOS-XR, EOS & NXOS

### PR DESCRIPTION
**Pull request by jejenone**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,9 +18,11 @@ script:
   - nosetests -v TestJunOSDriver:TestGetterJunOSDriver.test_get_interfaces
   - nosetests -v TestJunOSDriver:TestGetterJunOSDriver.test_get_interfaces_counters
   - nosetests -v TestJunOSDriver:TestGetterJunOSDriver.test_get_lldp_neighbors
-  - nosetests -v TestJunOSDriver:TestGetterJunOSDriver.test_get_lldp_neighbors_detail
```
Comment (mirceaulinic) [inline .travis.yml]: Also get_lldp_neighbors_details has some problems and will fix it tomorrow :(

**Comment by mirceaulinic** [comment]
All right, one more ready to be merged! :)

**Comment by mirceaulinic** [comment]
Since for get_arp_table made more sense to return a list (also the name "table" suggests that), we thought that it would be better to change the structure of the output for get_mac_adddress_table to list as well.

**Comment by jejenone** [comment]
@dbarrosop any blocker on that one ?

**Pull request merged**